dockers/docker-orchagent/wait_for_link.sh.j2

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+VLAN_TABLE_PREFIX="VLAN_TABLE"
+
+function wait_until_iface_ready
+{
+    TABLE_PREFIX=$1
+    IFACE=$2
+
+    echo "Waiting until interface $IFACE is ready..."
+
+    # Wait for the interface to come up
+    # (i.e., interface is present in STATE_DB and state is "ok")
+    while true; do
+        RESULT=$(sonic-db-cli STATE_DB HGET "${TABLE_PREFIX}|${IFACE}" "state" 2> /dev/null)
```

Comment:
During SWSS init, there is a cleanup of STATE_DB tables. Can you please confirm, this script is always executed after that cleanup?

Reply:
This script waits for `swssconfig.sh` to exit, and `swssconfig.sh` [waits for the `/ready` flag](https://github.com/Azure/sonic-buildimage/blob/a7f613038338093964a1d5299eb3e4f4fcbd0f0b/dockers/docker-orchagent/swssconfig.sh#L39) which is only created after the [DBs are cleared](https://github.com/Azure/sonic-buildimage/blob/a7f613038338093964a1d5299eb3e4f4fcbd0f0b/files/scripts/swss.sh#L175)

Reply:
So just to confirm, ndppd won't get started on a T1 topology with this change, right?